pkg/sqs/sqs.go

```diff
@@ -88,14 +87,7 @@ func (s *AWSSqsSource) Read(_ context.Context, readRequest sourcesdk.ReadRequest
 
 	ctx, cancel := context.WithTimeout(context.Background(), readRequest.TimeOut())
 	defer cancel()
-	// If we have un-acked data (data which is received but yet to be deleted from the queue
-	count, err := s.getApproximateMessageCount(approximateNumberOfMessagesNotVisible)
-	if err != nil {
-		log.Fatalf("error getting approximate number of messages not visible from queue %s", err)
-	}
-	if count > 0 {
-		return
-	}
+
```

Comment:
Line 100: `Fatal` means the container will crash, shall we just log the error and return?

Reply:
Is it possible to address the TODO in line 113?

Reply:
Line 96: should we use `readRequest.TimeOut()`?

Reply:
> Line 100: Fatal means the container will crash, shall we just log the error and return?

+1

> Line 96: should we use readRequest.TimeOut()?

+1

Reply:
@whynowy @KeranYang -As per aws docs - WaitTimeSeconds must be <= 0 and >= 20 if provided. References - so its difficult for us to use 		`WaitTimeSeconds:    readRequest.timeout(),` ,See references -https://docs.aws.amazon.com/powershell/latest/reference/?page=New-SQSQueue.html&tocid=New-SQSQueue

Reply:
> Is it possible to address the TODO in line 113?

Aws sqs doesn't provide any mechanism to get the time at which the message was sent to the queue ,removing this comment for now

Reply:
> Aws sqs doesn't provide any mechanism to get the time at which the message was sent to the queue.

Does the `SentTimestamp` attribute provide the sent time? See: https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_ReceiveMessage.html#SQS-ReceiveMessage-request-AttributeNames